composer-tips.md

```diff
@@ -0,0 +1,18 @@
+# Composer Tips
+
+### Composer update performance
+
+To get a better understanding of what is happening during a `composer update` the command should be run with `-vvv` which enabled a more verbose output.
+
+##### Resolving dependencies
+
+The snippet below shows the part of a composer update output which decides which packages should be installed. In cases where this takes a long time composer usually has to analyze a lot of packages.
+```
+Resolving dependencies through SAT
+Looking at all rules.
+Something's changed, looking at all rules again (pass #96)
+Dependency resolution completed in 129.785 seconds
+Analyzed 37751 packages to resolve dependencies
+Analyzed 1518987 rules to resolve dependencies
```

Comment:
Can you update this for Composer v2 depending on what it looks like there? Or maybe keep this and make it about v1 as this won't often be an issue on v2 anymore?